tutorials/smart-contracts/index.md

```diff
@@ -1,7 +1,23 @@
 ---
 title: Smart Contracts
-description: TODO
+description: Learn how to create, deploy, and manage smart contracts in the Polkadot ecosystem with detailed, step-by-step tutorials.
 template: index-page.html
 ---
 
-# Smart Contracts
\ No newline at end of file
+# Smart Contracts Tutorials
+
+Learn how to deploy and interact with smart contracts on Polkadot through practical, hands-on tutorials. These guides walk you through key deployment steps and best practices.
+
+## Tutorial Features
+
+- **Beginner to advanced** – suitable for developers at all experience levels
+- **Complete workflows** – covers the entire process from writing code to on-chain deployment
+- **Interactive examples** – follow along with working code that you can customize
+
+## Get Started
+
+Each tutorial provides detailed explanations, code examples, and deployment strategies to help you build confidently.
```

Comment:
I'd also move this up to the intro. This is a get started section and if you look at it as a standalone section, there are no CTAs here to get started.